### ISSUE TITLE
Contact images that are portrait-oriented are squashed as square thumbnails

### ISSUE THREAD

**Issue by acabal** [monicahq/monica#3543]
**Describe the bug**
When uploading a portrait-oriented image to contact, the square thumbnail is a squashed version of the portrait, instead of a correctly-proportioned crop.

On a locally-hosted instance of Monica v2.16.0 on Ubuntu 18.04, PHP 7.2.

**Screenshots**
For example, upload the following self-portrait of Albrecht Durer to a contact: 
![durer](https://user-images.githubusercontent.com/866144/73603859-f1380480-454d-11ea-9644-80f078333f07.jpg)

Result:
![portrait](https://user-images.githubusercontent.com/866144/73603865-057c0180-454e-11ea-8591-eae80be608fd.jpg)

Expected result:
The square thumbnail should be proportionally cropped so that the person doesn't appear squashed.

**Comment by cstrat** [comment]
Yeah I don't think the cropping works? I tried it on multiple photos uploaded and it does exactly what you've demonstrated above - just squishes the full size image.
